### ISSUE TITLE
Authorization to Upload Media to Records

### ISSUE THREAD

**Issue by Kmullineaux** [ArctosDB/arctos#8324]
Hello, I just received this error message. Can I please get authorization to upload media to records? Thanks!

ERROR_ID	F96269C8-453C-4825-86744A67E0D32B1A
ERROR_TYPE	unauthorized
ERROR_MESSAGE	not authorized
ERROR_DETAIL	You are not authorized to access this form.

**Comment by mkoo** [comment]
HI @Kmullineaux since you're listed as an administrative contact for NHSM I've added that permission
Please review:
* https://handbook.arctosdb.org/documentation/sharing-data-and-resources.html
* https://handbook.arctosdb.org/documentation/media.html
* https://handbook.arctosdb.org/how_to/How-to-Create-Media-Images.html

**Comment by mkoo** [comment]
@Kmullineaux let me know if you have any problems or questions with a new issue!

**Comment by Kmullineaux** [comment]
@mkoo thank you so much for your help!

**Comment by Kmullineaux** [comment]
@mkoo Hello! I am messaging with a follow-up question about uploading media to Tacc and attaching those URIs to Arctos records. I've followed the instructions here https://handbook.arctosdb.org/how_to/How-to-Upload-Media-to-TACC.html#large-batch-uploads and emailed Elizabeth Wommack ([arctos.communications@gmail.com] to request a project allocation. I attempted to upload files using the secure transfer program CyberDuck and they keep failing. Is there a way to tell if the allocation has been approved? Once it's approved, should I be able to bulk upload through CyberDuck?
